src/org/pentaho/reporting/platform/plugin/JobManager.java

```diff
@@ -18,40 +18,28 @@
 
 package org.pentaho.reporting.platform.plugin;
 
-import org.apache.commons.io.IOUtils;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.codehaus.jackson.map.ObjectMapper;
 import org.pentaho.platform.api.engine.IPentahoSession;
 import org.pentaho.platform.engine.core.system.PentahoSessionHolder;
 import org.pentaho.platform.engine.core.system.PentahoSystem;
-import org.pentaho.platform.util.RepositoryPathEncoder;
-import org.pentaho.platform.util.web.MimeHelper;
 import org.pentaho.reporting.platform.plugin.async.IAsyncReportState;
 import org.pentaho.reporting.platform.plugin.async.IPentahoAsyncExecutor;
 import org.pentaho.reporting.platform.plugin.async.PentahoAsyncExecutor;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
-import javax.ws.rs.WebApplicationException;
-import javax.ws.rs.core.CacheControl;
+import javax.ws.rs.*;
```

Comment:
[Pentaho Coding standards](http://wiki.pentaho.com/display/PEOpen/Code+Formatting+and+CheckStyle) say: No star imports. You can get the template for IntelliJ from [here](https://github.com/pentaho/pentaho-coding-standards)